### DISCUSSION TITLE
Can I use bubble button card to enable climate?

### DISCUSSION THREAD

**Discussion by pacemaker82** [Clooos/Bubble-Card#398] [General]
I set up a bubble button card on the car climate entity, but it doesnt do anything. I am assuming because to turn climate on you have to set the climate.set_hvac_mode or use the turn on service.

Is it possible to configure the bubble button card to do this with tap ON and tap OFF ?

The only work around was to do this: 

      - type: custom:bubble-card
        card_type: button
        entity: climate.none_climate
        show_state: true
        name: Climate
        icon: mdi:heat-wave
        tap_action:
          action: call-service
          service: climate.turn_on
        hold_action:
          action: call-service
          service: climate.turn_off

**Comment by Clooos** [comment]
Hi, this is possible with the custom button, take a look at this:

https://github.com/Clooos/Bubble-Card/?tab=readme-ov-file#custom-button

**Reply by pacemaker82** [reply]
Thanks - I guess I am missing something - it just seems like the regular button card - what am I missing thats different about the custom button?

**Reply by Clooos** [reply]
The custom button allow the tap actions to be on the whole button and not just on the icon, so you can choose the services you want to use on it.

**Reply by pacemaker82** [reply]
Thanks again, I guess something isnt connecting for me. I want to turn climate on and off with a single tap (I guess toggle? Tap button to turn on, tap button again to turn off). I created a work around above but that isnt what I actually want. Even with the custom button I would still need to do my work around no?

**Reply by Clooos** [reply]
I'm sorry I misunderstood your question, indeed you can't do more right now. But more cards and more customizations are planned, I can't wait to finally work on that!

**Reply by pacemaker82** [reply]
ok thank you!
